src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -218,6 +221,7 @@ function UserLayout(props: any): ReactElement {
               </div>
             </div>
             {navLinks.map((link: any, index: any) => (
+              <>
```

Comment:
I don't think there is need to have fragment as there is only one component i.e `NavLink`